core/http-auth-aws-crt/src/main/java/software/amazon/awssdk/http/auth/aws/crt/AwsCrtV4aHttpSigner.java

```diff
@@ -16,16 +16,16 @@
 package software.amazon.awssdk.http.auth.aws.crt;
 
 import software.amazon.awssdk.annotations.SdkPublicApi;
-import software.amazon.awssdk.http.auth.aws.crt.internal.DefaultAwsCrtV4aHttpSigner;
-import software.amazon.awssdk.http.auth.spi.HttpSigner;
+import software.amazon.awssdk.http.auth.aws.AwsV4aHttpSigner;
+import software.amazon.awssdk.http.auth.aws.crt.internal.signer.DefaultAwsCrtV4aHttpSigner;
 import software.amazon.awssdk.identity.spi.AwsCredentialsIdentity;
 
 /**
- * An {@link HttpSigner} that will sign a request using an AWS credentials ({@link AwsCredentialsIdentity}),
- * specifically for CRT.
+ * An {@link AwsV4aHttpSigner} that will sign a request using an AWS credentials ({@link AwsCredentialsIdentity}), specifically
+ * using CRT.
  */
 @SdkPublicApi
-public interface AwsCrtV4aHttpSigner extends HttpSigner<AwsCredentialsIdentity> {
+public interface AwsCrtV4aHttpSigner extends AwsV4aHttpSigner {
```

Comment:
now that we have the `AwsV4aHttpSigner` interface, why do we need the `AwsCrtV4aHttpSigner` interface too?

Reply:
We don't.